ui/pages/confirmations/components/confirm-subtitle/confirm-subtitle.js

```diff
@@ -1,28 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { useSelector } from 'react-redux';
 
 import { SECONDARY } from '../../../../helpers/constants/common';
 import {
   Color,
   TextVariant,
 } from '../../../../helpers/constants/design-system';
 import { isNFTAssetStandard } from '../../../../helpers/utils/transactions.util';
-import { getShouldShowFiat } from '../../../../selectors';
 import { useTransactionInfo } from '../../hooks/useTransactionInfo';
 import { Text } from '../../../../components/component-library';
 import UserPreferencedCurrencyDisplay from '../../../../components/app/user-preferenced-currency-display';
+import { useHideFiatForTestnet } from '../../../../hooks/useHideFiatForTestnet';
 
 const ConfirmSubTitle = ({
   txData,
   hexTransactionAmount,
   subtitleComponent,
   assetStandard,
 }) => {
-  const shouldShowFiat = useSelector(getShouldShowFiat);
   const { isNftTransfer } = useTransactionInfo(txData);
+  const hideFiatForTestNet = useHideFiatForTestnet();
 
-  if (!shouldShowFiat && !isNftTransfer && !isNFTAssetStandard(assetStandard)) {
```

Comment:
Do we not want to still check that fiat is enabled at all? Otherwise we will always show fiat if not on a testnet?

Also, if we've updated the `getShouldShowFiat` to check testnets, could we leave this code unchanged?